pkg/kv/metrics.go

```diff
@@ -31,6 +32,7 @@ type StoreMetricsWrapper struct {
 func (s *StoreMetricsWrapper) Get(ctx context.Context, partitionKey, key []byte) (*ValueWithPredicate, error) {
 	const operation = "Get"
 	timer := prometheus.NewTimer(requestDuration.WithLabelValues(s.StoreType, operation))
+	ctx = httputil.SetClientTrace(ctx, "kv")
```

Comment:
Same comments as for the "block" client-trace: I would prefer to include the type of the store in this string.

Reply:
Done.